x-pack/plugins/ml/common/types/file_datavisualizer.ts

```diff
@@ -19,6 +19,60 @@ export interface AnalysisResult {
   overrides?: FormattedOverrides;
 }
 
+type NumericType =
```

Comment:
The data plugin has these strings defined that we could import.
Annoyingly they defined them in their public location, and importing into common causes a linting error.
It seems a shame to redefine them.

Reply:
I've just noticed that these types aren't used at all. I think they should be removed and if we do need them in the future we can base them on the ones exported by the data plugin.